magda-web-client/src/Components/Common/DataPreviewMapOpenInNationalMapButton.tsx

```diff
@@ -78,11 +124,13 @@ class DataPreviewMapOpenInNationalMapButton extends Component<PropsType> {
                                 name: distribution?.title,
                                 type: "magda",
                                 recordId: distribution?.identifier,
-                                url: config.baseExternalUrl,
-                                id
+                                url: "https://dev.magda.io/", //config.baseExternalUrl,
```

Comment:
we probably don't need to overwrite this. I think UI preview will evaluate this value to `https://dev.magda.io/`

Reply:
Restored.